db/seeds.rb

```diff
@@ -13,34 +13,50 @@ def generate
-      @category2.items.create(title: "macciatto", description: "ryans drinks", price: 5.00, image: "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSuozcuZZYXrQpEJe0u1XOIl0g86R4ppXAvv3_yIyilkuX2F0wV")
-      @category2.items.create(title: "double espresso", description: "frothy deliciousness", price: 4.00, image: "https://encrypted-tbn1.gstatic.com/images?q=tbn:ANd9GcSSuvRAQORZqtyF1r2770nPSMGBoh_W8rGcArd-SowSv3kzIY5p")
+    @category1.items.create(title: "Fresh Brewed Coffee", description: "Black Coffee roasted and brewed in house", price:2, image: "https://encrypted-tbn2.gstatic.com/images?q=tbn:ANd9GcSgSHt_eNeOtwj7YfD2pzb936T-p967kHRdSVz0rFAijRxQtCP_Kg")
+    @category3.items.create(title: "Espresso shot", description: "A shot of espresso skillfully pulled by your barista", price:3, image: "https://encrypted-tbn3.gstatic.com/images?q=tbn:ANd9GcQobXI0QMDBTa-qs27YXoXzH1mwoAnlafJExoW3Y86sGoZ4Xo50Cg")
+    @category2.items.create(title: "Chai Tea Latte", description: "Aromic black tea served as a creamy sweat latte.", price: 5, image: "https://encrypted-tbn2.gstatic.com/images?q=tbn:ANd9GcSH-SgrF3bSunZIVfCuOtF2ENmJaDcNreEbvanX4CC1ivzVgfMC")
+    @category2.items.create(title: "Mocha Latte", description: "Hot chocolate and a shot of espresso brewed up in this tasty latte", price: 4, image: "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTkDBRWKJ5ZZShWt9U8RKQVXtObCbyF_KvQzI4ApXr9ral44EXT")
+    @category3.items.create(title: "Americano", description: "A smooth coffee that mixes shots of espress with hot water", price:2, image: "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRMNSxUtudEdAWOJWjG-Jb7MrR37jd1teNNFAdN3JA-SdMuji35")
+    @category3.items.create(title: "Cappucino", description: "Espresso with steamed, frothy milk and chocolate dusting.", price:4, image: "https://encrypted-tbn1.gstatic.com/images?q=tbn:ANd9GcRNSXFMdDggKNp-u3LkdJuUxuzXTXL21MgSOWms4NGQaW_T0Mlq")
+    @category4.items.create(title: "Tea", description: "our signature English Breakfast tea", price: 2, image: "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSuozcuZZYXrQpEJe0u1XOIl0g86R4ppXAvv3_yIyilkuX2F0wV")
+    @category3.items.create(title: "Macchiato", description: "two shots of espresso and some steamed milk", price: 3, image: "https://encrypted-tbn1.gstatic.com/images?q=tbn:ANd9GcSSuvRAQORZqtyF1r2770nPSMGBoh_W8rGcArd-SowSv3kzIY5p")
 
     puts "items created."
   end
 
   def create_admin
-    User.create(username: "admin", password: "pw", role: 1)
+    User.create(username: "admin", password: "pw", role: 1, address: "adminland", city: "Denver", state: "CO", zip: 83020)
 
     puts "admin created"
   end
 
   def create_registered_user
-    User.create(username: "Joe", password: "secret", role: 0)
+    @user1 =  User.create(username: "ryan", password: "waffles", role: 0, address: "1111 Maple Rd", city: "Denver", state: "CO", zip: 82020)
```

Comment:
Line is too long. [140/80]<br>Separate every 3 digits in the integer portion of a number with underscores(_).